settings.gradle

```diff
@@ -0,0 +1,26 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+        mavenCentral()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
```

Comment:
Just to double check: I'm assuming this is still necessary even though it's google() and mavenCentral() are included in the block above?

Reply:
This came by default when I initialized the repository. Whenever I remove them it fails to properly gradle.